src/Language/PureScript/Sugar/TypeClasses/Deriving.hs

```diff
@@ -558,39 +558,26 @@ deriveNewtype
   :: forall m
    . (MonadError MultipleErrors m, MonadSupply m)
   => SourceSpan
-  -> ModuleName
   -> SynonymMap
   -> KindMap
   -> [Declaration]
   -> ProperName 'TypeName
   -> [SourceType]
   -> SourceType
   -> m ([Declaration], SourceType)
```

Comment:
We could return a `m SourceType` here, since the class has no methods anymore.

Reply:
Oh, good point. Done.